### PR TITLE
Add section regarding polkadot PRs.

### PR DESCRIPTION
**Pull request by gnunicorn**
Subsection to remind everyone about updating polkadot PRs, too.

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -45,6 +45,21 @@ When reviewing a pull request, the end-goal is to suggest useful changes to the
...
 . It does not fit well with some other contributors' longer-term vision for the project.
 
+=== Updating Polkadot as well
+
+If your PR changes the external APIs or interfaces used by Polkadot, **a corresponding PR on Polkadot must** be submitted as well. If you tagged the PR with `breaksapi` or `breaksconsensus` this is most certainly the case, in all other cases check for it by running step 1. 
+
+To update a corresponding Polkadot PR:
+
+0. pull lastet Polkadot master (or clone it, if you haven't yet)
+1. replace `pokadot-master` in all `Cargo.toml` with the name of the PR-branch (and to your repo, if the branch is not on mainline) - commit this change.
```
Comment (rphmeier) [inline CONTRIBUTING.adoc]:
```suggestion
1. replace `polkadot-master` in all `Cargo.toml` with the name of the PR-branch (and to your repo, if the branch is not on mainline) - commit this change.
```

Comment (bkchr) [inline CONTRIBUTING.adoc]: You could add this snippet:
```
find . -name "Cargo.toml" -exec sed -i "s/polkadot-master/my-branch/g" {} +
```

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -45,6 +45,21 @@ When reviewing a pull request, the end-goal is to suggest useful changes to the
 . There exists a somewhat cleaner/better/faster way of accomplishing the same feature/fix.
 . It does not fit well with some other contributors' longer-term vision for the project.
 
+=== Updating Polkadot as well
+
+If your PR changes the external APIs or interfaces used by Polkadot, **a corresponding PR on Polkadot must** be submitted as well. If you tagged the PR with `breaksapi` or `breaksconsensus` this is most certainly the case, in all other cases check for it by running step 1. 
+
+To update a corresponding Polkadot PR:
+
+0. pull lastet Polkadot master (or clone it, if you haven't yet)
```
Comment (bkchr) [inline CONTRIBUTING.adoc]:
```suggestion
0. Pull lastet Polkadot master (or clone it, if you haven't yet).
```

File: CONTRIBUTING.adoc [hunk 3]
```diff
@@ -45,6 +45,21 @@ When reviewing a pull request, the end-goal is to suggest useful changes to the
...
+1. replace `pokadot-master` in all `Cargo.toml` with the name of the PR-branch (and to your repo, if the branch is not on mainline) - commit this change.
+2. make the changes required to pass the build again
```
Comment (bkchr) [inline CONTRIBUTING.adoc]:
```suggestion
2. Make the changes required to pass the build again.
```

File: CONTRIBUTING.adoc [hunk 4]
```diff
@@ -45,6 +45,21 @@ When reviewing a pull request, the end-goal is to suggest useful changes to the
...
+3. Submit all this as a PR against the Polkadot Repo, link that new PR in the existing substrate PR for reference
+4. Wait for reviews on both
+5. Once both PRs have been green lit, merge the Substrate PR into master and sync the changes onto the `polkadot-master` on mainline (push it). Now revert that first commit in your Polkadot PR, reverting the name back to `polkadot-master`, push and wait for the CI to confirm all is fine. Then merge the Polkadot PR.
```
[the remark this answers is not in the document]
Comment (gnunicorn) [reply CONTRIBUTING.adoc]: Yeah, I was wondering the same thing at first, but then thought: a) this is currently attempting to solve the issue for internal people, and b) whoever is merging clearly has that responsibility. Considering that a Polkadot PR is linked to the issue it should be clear where to do this...

File: CONTRIBUTING.adoc [hunk 5]
```diff
@@ -45,6 +45,21 @@ When reviewing a pull request, the end-goal is to suggest useful changes to the
...
 . It does not fit well with some other contributors' longer-term vision for the project.
 
+=== Updating Polkadot as well
+
+If your PR changes the external APIs or interfaces used by Polkadot, **a corresponding PR on Polkadot must** be submitted as well. If you tagged the PR with `breaksapi` or `breaksconsensus` this is most certainly the case, in all other cases check for it by running step 1. 
+
+To update a corresponding Polkadot PR:
+
+0. Pull lastet Polkadot master (or clone it, if you haven't yet).
+1. replace `polkadot-master` in all `Cargo.toml` with the name of the PR-branch (and to your repo, if the branch is not on mainline) - commit this change.
```
Comment (gnunicorn) [inline CONTRIBUTING.adoc]:
```suggestion
1. Replace `polkadot-master` in all `Cargo.toml` with the name of the PR-branch - e.g. by running `find . -name "Cargo.toml" -exec sed -i "s/polkadot-master/my-branch/g" {}` - (and to your repo, if the branch is not on mainline); Commit this change 
```

**Pull request merged**